java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -2003,4 +2003,21 @@ void testStringTitlize() {
       assertColumnsAreEqual(expected, result);
     }
   }
+
+  @Test
+  void testNansToNulls() {
+    try (ColumnVector cvFloat = ColumnVector.fromBoxedFloats(1.2f, Float.POSITIVE_INFINITY,
```

Comment:
I recommend moving the list of floats outside of this try-with-resources block to make it much easier to read, e.g.:
```java
  Float[] inputFloats = new Float[] { 1.2f, Float.POSITIVE_INFINITY, ... };
  Float[] expectedFloats = new Float[] { 1.2f, ... };
  ...
  try(ColumnVector cvFloat = ColumnVector.fromBoxedFloats(inputFloats);
      ColumnVector expectedFloat = ColumnVector.fromBoxedFloats(expectedFloats);
        ...) {
```